src/components/select2/AuiSelect2Single.vue

```diff
@@ -15,7 +15,8 @@
       placeholder: String,
       query: Function,
       value: String,
-      width: String
+      width: String,
+      optionRenderer: Function
```

Comment:
I try to keep original select2 API names here to make it more clear so this should be named `formatResult` perhaps.

Reply:
Yes, I agree, I will rename it